### PR TITLE
feat: modernise generic onboarding

### PR DESCRIPTION
**Pull request by Nisha1306**
Solves https://github.com/AtB-AS/kundevendt/issues/1916 
Figma Link: https://www.figma.com/file/zdZwvobgpEWSagKt0tderx/App-1.2?node-id=5222%3A21656

It has been discussed with @cbrevik to not upgrade the illustrations.

### CODE REVIEW

**Comment by monopoint** [comment]
Wrong color in status bar (android)
image

File: src/screens/Onboarding/index.tsx [hunk 1]
```diff
@@ -13,41 +17,33 @@ export type OnboardingStackParams = {
...
-      {/*      name="ConfirmCodeInOnboarding"*/}
-      {/*      component={ConfirmCodeInOnboarding}*/}
-      {/*    />*/}
-      {/*    <Stack.Screen name="SkipLoginWarning" component={SkipLoginWarning} />*/}
-      {/*  </>*/}
-      {/*) : (*/}
-      <>
-        <Stack.Screen
+    <SafeAreaView style={styles.container}>
+      <Tab.Navigator
```
Comment (monopoint) [inline src/screens/Onboarding/index.tsx:27]: add
```
<StatusBar
    backgroundColor={theme.static.background[themeColor].background}
/>
```
for android

**Review by monopoint — CHANGES_REQUESTED** [review]
some feedback

File: src/screens/Onboarding/index.tsx [hunk 2]
```diff
@@ -13,41 +17,33 @@ export type OnboardingStackParams = {
...
+      >
+        <Tab.Screen
           name="WelcomeScreenWithoutLogin"
           component={WelcomeScreenWithoutLogin}
         />
-        <Stack.Screen name="IntercomInfo" component={IntercomInfo} />
-      </>
-      {/*)}*/}
-    </Stack.Navigator>
+        <Tab.Screen name="IntercomInfo" component={IntercomInfo} />
```
[the remark this answers is not in the document]
Comment (Nisha1306) [reply src/screens/Onboarding/index.tsx:38]: We are not really sure about this. This task was to make the look and feel of general onboarding to be in sync with Mobile token onboarding. So not sure why we have the fourth screen is here. Also we do not have any onboarding for testpilot like the second screen, so we have ignored that as well. Have also discussed with @cbrevik  and concluded on adding the location screen separately if its needed.  Issue added https://github.com/AtB-AS/kundevendt/issues/2031

**Review by monopoint — APPROVED** [review]
:+1:

**Pull request merged**